### PR TITLE
Allow hiding loader on Heatmap and add story for live data

### PR DESCRIPTION
**Pull request by axelboc**

### CODE REVIEW

File: src/stories/HeatmapVis.stories.tsx [hunk 1]
```diff
@@ -75,6 +76,28 @@ NoGrid.args = {
...
+      .slice(0)
+      .sort(() => 0.5 - Math.random());
+
+    setShuffledArray(
+      ndarray<number>(shuffledValues, dataset.shape.dims).transpose(1, 0)
+    );
+  }, 5000);
+
+  return <HeatmapVis {...args} dataArray={shuffledArray} />;
+};
```
Comment (axelboc) [inline src/stories/HeatmapVis.stories.tsx:93]: Every 5000ms, I shuffle the array.

File: src/stories/HeatmapVis.stories.tsx [hunk 2]
```diff
@@ -75,6 +76,28 @@ NoGrid.args = {
   showGrid: false,
 };
 
+export const LiveData: Story<HeatmapVisProps> = (args): ReactElement => {
```
Comment (loichuder) [inline src/stories/HeatmapVis.stories.tsx:79]: It would be nice to mention the loader in the story name. `LiveDataWithoutLoader` ?

**Review by loichuder — APPROVED** [review]
:+1:

File: src/stories/HeatmapVis.stories.tsx (hunk 2 above)
Comment (axelboc) [reply src/stories/HeatmapVis.stories.tsx]: nice one 👍

**Pull request merged**